tests/__init__.py

```
@@ -49,48 +49,22 @@
-
-try:
-	if 'functools' not in sys.modules:
-		import functools
-	else:  # pragma: no branch
-		functools = sys.modules["""functools"""]
-except Exception as err:  # pragma: no branch
-	raise ModuleNotFoundError("[CWE-440] functools Failed to import.") from err
+	import os
+	import unittest
+except ImportError as baton:  # pragma: no branch
+	raise ModuleNotFoundError("[CWE-440] Module failed to import.") from baton
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)